src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -186,6 +186,8 @@ protected override bool OnShutdown(int timeoutMilliseconds)
             this.shutdownDrainTarget = this.circularBuffer.AddedCount;
             this.shutdownTrigger.Set();
 
+            OpenTelemetrySdkEventSource.Log.DroppedExportProcessorItems(nameof(BatchExportProcessor<T>), typeof(T).Name, this.exporter.GetType().Name, this.droppedCount);
```

Comment:
if you just do "this.GetType().Name" and "this.exporter.GetType().Name", we get what we need.